README.md

```diff
@@ -1,97 +1,86 @@
 # CovidTracker
 
-## Server
-
-### Initial setup
-
-Make a copy of `.env.example`, rename it to `.env`, and fill out any missing secret values (ask someone on the team if you don't know what the value should be)
-
-By default, the server will be exposed on `localhost:8080` if you have a port conflict with that change the docker-compose entry port for the server service from "8080:80" to "<new-port>:80"
-
-### Running the API
-
-Create the docker container for the server and database with
-```
-# Windows
-
-
-# Linux
-sudo docker-compose up -d server
-```
-This will start
-- API on `localhost:8080`
-- PostgreSQL database on `localhost:5432`
-
-### Running tests
+## Getting Started
+
+### Prerequisites
+
+Before you continue, ensure you have installed the following requirements:
+- [Docker](https://www.docker.com/)
+- [pgAdmin](https://www.pgadmin.org/)
+- [npm](https://www.npmjs.com/)
+
+### Setup
+
+1. Clone the repository
+    ```sh
+    $ git clone https://github.com/SOEN-390-Mini-Cap/CovidTracker.git
+    ```
+2. Copy `.env.sample`, rename to `.env` and edit the following lines
+    ```
+    TBD
+    ```
+3. Install all dependencies
+    ```sh
+    $ cd server
+    $ npm i
+    ```
+4. Create the docker container
+    ```sh
+    $ docker-compose up -d server
+    ```
+5. Connect to the database on `localhost:5432` in pgAdmin
+
+You can now access the server at http://localhost:8080
+
+**Note:** 
+
+By default, the server and database will be exposed on `localhost:8080` and `localhost:5432`, respectively. If you have a port conflict with either one, change their respective docker-compose entry ports as follows:
+- Server from `8080:80` to `<new-port>:80`
+- Database from `5432:5432` to `<new-port>:5432`.
+
+### Running the Tests
 
 #### Locally
 
-1. Install server dependencies inside `/server` folder
-```
-npm i
+Run all tests with the following command in the `server` directory
 ```
-2. Run unit tests
-```
-npm run test:unit
+$ npm run test:unit
 ```
 
 #### Docker
 
-1. Start docker container with docker-compose
-```
-# Windows
-
-
-# Linux
-sudo docker-compose up -d server
-```
-2. Exec into the docker container with
-```
-# Windows
-
-
-# Linux
-sudo docker-compose exec server sh
-```
-3. Run unit tests
-```
-npm run test:unit
-```
-
-### Linting and formatting
-
-1. Install server dependencies inside `/server` folder
-```
-npm i
-```
-2. Run lint and formatter
-```
-npm run lint:fix
-```
-
-### Building for production
-
-To build the server for production we will build the docker image with the production target
-
-```
-# Windows
-
-
-# Linux
-sudo docker build --target production -t covid-tracker-server-prod .
-```
-
-We can then create a container from the production image
-
-```
-# Windows
-
-
-# Linux
-sudo docker run -p 8080:80 covid-tracker-server-prod
-```
-
-## Maintainers
+1. Start the docker container
+    ```sh
+    $ docker-compose start server db
```

Comment:
The `docker-compose.yml` file specifies dependencies so you only have to specify to bring up the `server` service.
Also you will want to use `up` instead of `start`, `start` command will bring up an existing container instead of rebuilding it and you almost always want to rebuild.

```suggestion
    $ docker-compose up server
```

Reply:
Makes sense.